js/main.js

```diff
@@ -820,6 +826,17 @@ require([
 				point.geometry.latitude = mapObjects.points[i].pos[1];
 				point.geometry.longitude = mapObjects.points[i].pos[0];
 				renderLayer.add(point);
+
+				if(mapObjects.points[i].showname)
+				{
+					let name_text = new Graphic(GraphicsLibrary.distanceLabel);
+					name_text.geometry.longitude = mapObjects.points[i].pos[0];
+					let shift = 0.1;
```

Comment:
That shift is a bit of a problem since it's not zoom invariant and results in the text sliding offscreen when really zoomed in: 
![image](https://github.com/MoffKalast/Sailwind-Map/assets/9977799/08ddb8f9-eabf-4303-bc51-8baaa451b58e)

I would suggest using [the offset param](https://github.com/MoffKalast/Sailwind-Map/blob/00bde360ede96948b6941b1fa942067988fcec40/js/graphics.js#L277) instead.